pydarn/plotting/power.py

```diff
@@ -0,0 +1,146 @@
+#  Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+#  Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+from pydarn import SuperDARNRadars, exceptions, RTP
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_lag0
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    @classmethod
+    def plot_lag0(cls, records: List[dict], beam_num: int = 0,
+                  compare: bool = True, frequency: float = 11000,
+                  statistical_calc: object = np.mean):
+
+        """
+        This function will plot lag 0 power as a function of time.
+
+
+        Extra  Details
+        --------------
+        If the records that are input correspond to politescan data then this
+        can be a useful method for determining background interference
+        variations and patterns.
+
+        If you wish to compare the background interference associated with two
+        different frequencies then let compare=true. The frequencies will be
+        organized by the 'frequency' input. For example, if politescan ran
+        with 10.3 and 12.2 MHz then you could let 'frequency' equal 11000.
+        All records with tfreq below 11000 will be separated from those records
+        with tfreq above 11000. If compare = False then frequency is simply
+        the frequency that politescan ran with i.e 12.2 MHz exclusively.
+
+        Future Work
+        ___________
+        Allow for multi-beam comparison to get a directional
+        sense of the interference
+
+
+        statistical_calcs
```

Comment:
The heading looks wrong here?
```suggestion
        Parameters
```

Reply:
Also, I think Parameters goes above some of the other docstring headings in the numpydoc style guide, and Extra Details could be renamed to 'Notes'